gulpfile.js

```diff
@@ -25,7 +25,7 @@ gulp.task('run', function () {
   nodemon({
     script: 'src/index.js',
     ext: 'js',
-    env: { 'DEBUG': '*' },
+    /** env: { 'DEBUG': '*' }, **/
```

Comment:
what happens when this is un/commented? maybe a comment explaining that would be useful